content/docs/strict-mode.md

```diff
@@ -116,6 +116,10 @@ At first glance, this code might not seem problematic. But if `SharedApplication
 
 By intentionally double-invoking methods like the component constructor, strict mode makes patterns like this easier to spot.
 
+> Note:
+>
+> Currently, React automatically [modifies](https://github.com/facebook/react/pull/18547) the console methods like `console.log()` to silence the logs in the second call to lifecycle functions. However, it may cause undesired behavior in certain cases, where [a workaround can be used](https://github.com/facebook/react/issues/20090#issuecomment-715927125).
```

Comment:
```suggestion
> Currently, React automatically modifies the console methods like `console.log()` to silence the logs in the second call to lifecycle functions. However, it may cause undesired behavior in certain cases, where [a workaround can be used](https://github.com/facebook/react/issues/20090#issuecomment-715927125).
```

The implementation isn't important for documentation as far as I can tell. It may even be misleading in case code moves around.

```suggestion
> Currently, React automatically modifies the console methods like `console.log()` to silence the logs in the second call to lifecycle functions. However, it may cause undesired behavior in certain cases where [a workaround can be used](https://github.com/facebook/react/issues/20090#issuecomment-715927125).
```
Not sure about that comma. Seems off to me but if you're sure then keep it.